contents.json

```diff
@@ -2109,6 +2109,11 @@
       "description": "A simple static table views for iOS.",
       "homepage": "https://github.com/venmo/Static"
     }, {
+      "title": "FlexLayout",
+      "category": "layout",
+      "description": "FlexLayout adds a nice Swift interface to the highly optimized facebook/yoga flexbox implementation. Concise, intuitive & chainable syntax. Works perfectly with PinLayout.",
```

Comment:
please remove "Swift" from the description

Reply:
The branch has been updated. Thanks